01week/resume/index.html

```diff
@@ -1,13 +1,86 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="stylesheet" href="resumesheet.css" type="text/css">
     <title>Cameron Harmon ~ Resume</title>
   </head>
   <body>
-    <h1>Cameron Harmon</h1>
-    <h2>Developer in Austin, TX looking for work.</h2>
-    <p>
-      I like to code and ride my bicycle Downtown
-    </p>
+    <nav>
+      <a href="#">About</a> |
+      <a href="#">Experience</a> |
+      <a href="#">Education</a> |
+      <a href="#">Contact</a>
+    </nav>
+    <header>
+      <h1 id="name">Cameron Harmon</h1>
+      <p>Web Developer from Austin, TX looking for work.</p>
+    </header>
+
+    <main>
+        <section id="About">
+          <p>I have spent years solving a variety of problems for people ranging from sleep deprivation to employee efficiency and morale.
+            Recently I have shifted my focus from helping corporations to helping the community. I use Web Development skills to build
```

Comment:
Can follow through on code once it gets to long by starting it on new line underneath.  This allows for neater code.